generator/resources/ec2_mac_test_matrix.json

```diff
@@ -1,2 +1,26 @@
 [
+      {
+        "os": "macOS Big Sur Arm64",
+        "instanceType":"mac1.metal",
+        "ami": "amzn-ec2-macos-11.*",
+        "arc": "arm64"
+      },
+      {
+        "os": "macOS Big Sur Amd64",
+        "instanceType":"mac1.metal",
+        "ami": "amzn-ec2-macos-11.*",
+        "arc": "amd64"
+      },
+      {
+        "os": "macOS Ventura Arm64",
+        "instanceType":"mac2.metal",
+        "ami": "amzn-ec2-macos-13.*",
+        "arc": "arm64"
+      },
+      {
+        "os": "macOS Ventura Amd64",
```

Comment:
Is there no mac os 12?

Reply:
No thought. We only test a subset of MacOs.